cluster/ci/config/prow/config.yaml

```diff
@@ -2380,6 +2380,26 @@ presubmits:
         args:
           - ./hack/tf-lint.sh
 
+  - name: go-vet
```

Comment:
nit -- this is usually ci-pull-$org-$repo-$name

Reply:
Done